.circleci/config.yml

```diff
@@ -110,6 +114,7 @@ jobs:
           source ~/env/bin/activate
           export PATH=$JAVA_HOME/bin:$PATH
           pip3 install --upgrade -r ~/cassandra-dtest/requirements.txt
+          pip3 uninstall -y cqlsh
```

Comment:
why?

Reply:
config.yml is supposed to be config.yml.LOWER but looks like it want done properly in py3 patch, since I registered the files this got updated